galgebra/printer.py

```diff
@@ -1130,12 +1129,6 @@ def Fmt(obj, fmt=0):
                 #cell.title = None
                 latex_cell = latex(cell)
                 latex_cell = latex_cell.replace('\n', ' ')
-                latex_cell= latex_cell.replace(r'\begin{equation*}', ' ')
-                latex_cell= latex_cell.replace(r'\end{equation*}', ' ')
```

Comment:
These could never happen, none of our objects or sympy's objects emit `equation` environments.